src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -805,15 +871,15 @@ TEST_F(MessageHelperTest,
   images[1][strings::value] = "image\\n";
   // Method call
   mobile_apis::Result::eType result = MessageHelper::VerifyImageFiles(
-      images, appSharedMock, mock_application_manager);
+      images, app_shared_mock, mock_application_manager);
```

Comment:
@indlu it would be eisier to review if you split this commit to 2 commits :
 - logic
- formating